src/index.ts

```diff
@@ -115,6 +115,19 @@ export const main = async (options: Options): Promise<void> => {
 				roleArn: roleArn,
 				webIdentityToken: idToken,
 			}),
+
+			/*
+      Increase the timeout. Since updating `@aws-sdk/client-s3` to v3.717.0 we've seen an increase of timeouts:
+      > RequestTimeout: Your socket connection to the server was not read from or written to within the timeout period. Idle connections will be closed.
+
+      See https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/node-configuring-maxsockets.html
+       */
+			requestHandler: {
+				requestTimeout: 10 * 1000, // 10 seconds
+			},
+
+			// Enable AWS SDK logging if the action is run in debug mode
+			...(core.isDebug() && { logger: { ...console } }),
```

Comment:
See https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/logging-sdk-calls.html.